src/components/MewOverlay/MewOverlay.vue

```diff
@@ -1,50 +1,91 @@
+  <!-- Mew Overlay -->
+  <!-- ===================================================================================== -->
   <v-bottom-sheet
-    :value="isOverlayShown" 
+    :value="isOverlayShown"
     :fullscreen="true"
     persistent
+    content-class="mew-overlay"
   >
-    <v-sheet 
+    <v-sheet
       height="100%"
       color="emerald100"
       class="mew-overlay-container d-flex align-center flex-column"
     >
-      <!--
-  =====================================================================================
-    Mew Overlay action fab buttons (back and close)
-  =====================================================================================
-  -->
+      <div
+        v-if="isMobile"
+        class="mobile-title-block pa-3 d-flex align-center justify-space-between full-width"
+      >
+        <div style="min-width: 36px">
+          <v-btn
+            v-if="back"
+            icon
+            color="textBlack2"
+            @click="back"
+          >
+            <v-icon size="24">
+              mdi-arrow-left
+            </v-icon>
+          </v-btn>
+        </div>
+        <h3
+          class="titlePrimary--text mx-4 text-center"
+          style="line-height: 21px"
+        >
+          {{ title }}
+        </h3>
+        <div style="min-width: 36px">
+          <v-btn
+            v-if="close"
+            icon
+            color="textBlack2"
+            @click="close"
+          >
+            <v-icon
+              size="24"
+              color="textBlack2"
+            >
+              mdi-close
+            </v-icon>
+          </v-btn>
+        </div>
+      </div>
+
+      <!-- ===================================================================================== -->
+      <!-- Mew Overlay action fab buttons (back and close) -->
+      <!-- ===================================================================================== -->
       <v-btn
-        v-if="back"
+        v-if="back && !isMobile"
         top
         left
         absolute
         text
         color="textBlack2"
-        @click="back"
-        :class="['d-flex action-btn align-center pa-3', isMobile ? 'mt-3 ml-n1 mobile-btn' : 'mt-4 ml-1']"
+        :class="[
+          'd-flex action-btn align-center pa-3',
+          isMobile ? 'mt-3 ml-n1 mobile-btn' : 'mt-4 ml-1'
+        ]"
         fab
+        @click="back"
       >
-        <v-icon
-          size="24"
-        >
+        <v-icon size="24">
           mdi-arrow-left
         </v-icon>
       </v-btn>
       <v-btn
+        v-if="close && !isMobile"
         fab
         top
         right
         absolute
         text
         color="textBlack2"
-        :class="['d-flex action-btn align-center pa-3', , isMobile ? 'mt-3 mr-n1 mobile-btn' : 'mt-4 mr-1']"
-        v-if="close"
+        :class="[
+          'd-flex action-btn align-center pa-3',
+          ,
+          isMobile ? 'mt-3 mr-n1 mobile-btn' : 'mt-4 mr-1'
```

Comment:
is this conditional necessary? isMobile will always be false here since v-if requires isMobile to be false.

Reply:
Fixed!